src/qibocal/protocols/characterization/qutrit_classification.py

```diff
@@ -0,0 +1,189 @@
+            )
+            sequence.add(ro_pulses[qubit][-1])
+
+    # create a DataUnits object to store the results
+    data = QutritClassificationData(
+        nshots=params.nshots,
+        classifiers_list=params.classifiers_list,
+        classifiers_hpars=hpars,
+        savedir=params.savedir,
+    )
+    states_results = []
+    for sequence in states_sequences:
+        states_results.append(
+            platform.execute_pulse_sequence(
+                sequence,
+                ExecutionParameters(
+                    nshots=params.nshots,
+                    relaxation_time=params.relaxation_time,
+                    acquisition_type=AcquisitionType.INTEGRATION,
+                ),
+            )
+        )
+
+    for qubit in qubits:
+        for state, state_result in enumerate(states_results):
+            result = state_result[ro_pulses[qubit][state].serial]
+            data.register_qubit(
+                ClassificationType,
+                (qubit),
+                dict(
+                    state=[state] * params.nshots,
+                    i=result.voltage_i,
+                    q=result.voltage_q,
+                ),
+            )
+
+    return data
+
+
+def _fit(data: QutritClassificationData) -> SingleShotClassificationResults:
+    qubits = data.qubits
+
+    benchmark_tables = {}
+    models_dict = {}
+    y_tests = {}
+    x_tests = {}
+    hpars = {}
+    y_test_predict = {}
+    grid_preds_dict = {}
+    for qubit in qubits:
+        qubit_data = data.data[qubit]
+        benchmark_table, y_test, x_test, models, names, hpars_list = run.train_qubit(
+            data, qubit
+        )
+        benchmark_tables[qubit] = benchmark_table.values.tolist()
+        models_dict[qubit] = models
+        y_tests[qubit] = y_test.tolist()
+        x_tests[qubit] = x_test.tolist()
+        hpars[qubit] = {}
+        y_preds = []
+        grid_preds = []
+
+        grid = evaluate_grid(qubit_data)
+        for i, model_name in enumerate(names):
+            hpars[qubit][model_name] = hpars_list[i]
+            try:
+                y_preds.append(models[i].predict_proba(x_test)[:, 1].tolist())
+            except AttributeError:
+                y_preds.append(models[i].predict(x_test).tolist())
+            grid_preds.append(
+                np.round(np.reshape(models[i].predict(grid), (MESH_SIZE, MESH_SIZE)))
+                .astype(np.int64)
+                .tolist()
+            )
+        y_test_predict[qubit] = y_preds
+        grid_preds_dict[qubit] = grid_preds
+    return SingleShotClassificationResults(
+        benchmark_table=benchmark_tables,
+        names=names,
+        classifiers_hpars=hpars,
+        models=models_dict,
+        savedir=data.savedir,
+        y_preds=y_test_predict,
+        grid_preds=grid_preds_dict,
+    )
+
+
+def _plot(data: QutritClassificationData, qubit, fit: SingleShotClassificationResults):
+    figures = plot_results(data, qubit, 3, fit)
+    fitting_report = None
```

Comment:
```suggestion
    fitting_report = ""
```
Small fix I found after running on hardware